schedule/yast/autoyast_systemd_timesync.yaml

```diff
@@ -0,0 +1,28 @@
+name:       autoyast_systemd_timesync
+description:   >
+  Test performs autoyast installation where profile dictates that ntp is configured for one time synchronization.
+  Following, checks for the use of systemd timesync instead of cron, validates configuration and functionality.
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_systemd_timesync.xml
+  AUTOYAST_PREPARE_PROFILE: 1
```

Comment:
here you miss AYTOYAST_CONFIRM. this is used in the autoyast/installation.pm and you should see something like [this](http://aquarius.suse.cz/tests/2417#step/installation/3). obviously doesnt break the test without but it would add in coverage and maybe for other reasons that i am not familiar. Consider this if it is not a problem or it doesnt break anything

Reply:
I have used in the xml profile: 
<confirm config:type="boolean">false</confirm>
in order to avoid installation failure.